### ISSUE TITLE
Languages menu in Orange navbar burger menu : design + accessibility

### ISSUE THREAD

**Issue by isabellechanclou** [Orange-OpenSource/Orange-Boosted-Bootstrap#2514]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

**Design issue**:
In the DSM + in Boosted docs and examples, specifications for adding a language menu in a supra bar positioned above the Orange navbar component is documented for desktop screens. Now, coming to the corresponding mobile layout, it's not that clear:
- In the Orange navbar "[Responsive behavior](https://boosted.orange.com/docs/5.3/components/orange-navbar/#responsive-behavior)" section, we can see that the languages menu from the supra bar lands in the burger menu, all languages in a same line.

    ![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/282b9189-4cd0-486b-9171-c084268cdb11)

- In the Orange navbar, we can find dropdown menus (for choosing a version of Boosted, for choosing dark or light mode). In the burger menu, they still are displayed as dropdowns.

    ![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/49dd10de-1f68-4656-83e0-036b73a0ddbb)

So, we could wonder if in small screen how the language menu should be display in the burger menu between these two possibilities:
1- 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/91d90330-7dec-4dba-a247-fc962d638d30)

2- 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/d937310a-4637-4e1b-93f3-5bc6c07de02d)
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/c35339bf-e6f3-45be-92ed-3a0804ed237d)

- Moreover, confusion is added by the different ways encountered on different Orange websites for displaying languages. Quite often they are displayed as a dropdown or a select component in the Orange navbar (and not in the supra bar) for desktop, and for small screens, it can stay the same as the desktop display or be i then burger menu, all languages on a same line.

=> So, the rule to apply for the display of languages in desktop and in small screen would benefit to be more precised, indicated and illustrating what is/are the allowed variant(s) in the DSM and in Boosted.

**Accessibility issue**:
- In the burger menu, with the languages displayed in the same line, the selected language is indicated only by its color (orange instead of white), which is not enough regarding accessibility. There should be an alternative indicator for user who don't perceive colors properly. Proposal of correction : adding an orange line below the selected language:

    ![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/89514509/f57389fd-a884-45a5-acf8-1d1032b6cbd0)

### Reduced test cases

N.A.

### What operating system(s) are you seeing the problem on?

Windows, macOS

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Boosted are you using?

latest v5.3.x

**Comment by MaxLardenois** [comment]
Boosted only, won't fix
